### PR TITLE
Support scala 3.0.0-M3

### PR DESCRIPTION
**Pull request by bplommer**

### CODE REVIEW

File: core/src/test/scala/com/dimafeng/testcontainers/BaseSpec.scala [hunk 1]
```diff
@@ -5,11 +5,12 @@ import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatestplus.mockito.MockitoSugar
+import scala.reflect.ClassTag
 
-abstract class BaseSpec[T: Manifest]
+abstract class BaseSpec[T: ClassTag]
```
Comment (bplommer) [inline core/src/test/scala/com/dimafeng/testcontainers/BaseSpec.scala:10]: this breaks binary compat

**Comment by dimafeng** [comment]
@bplommer thank you for this contribution! 🚀 I'll review this over this week/weekend. 

@LMnet, would be great if you could take a look too.

File: core/src/test/scala/com/dimafeng/testcontainers/BaseSpec.scala (hunk 1 above)
Comment (LMnet) [reply core/src/test/scala/com/dimafeng/testcontainers/BaseSpec.scala:10]: It's in tests, so I don't think that this is a problem.

**Comment by LMnet** [comment]
I'm +1 on this.
Great work! Thanks @bplommer

**Pull request merged**